src/registrar/views/domain.py

```diff
@@ -653,12 +653,30 @@ def _send_domain_invitation_email(self, email: str, add_success=True):
         # created a new invitation in the database, so send an email
         domainInfoResults = DomainInformation.objects.filter(domain=self.object)
         domainInfo = domainInfoResults.first()
-        first = ""
-        last = ""
-        if domainInfo is not None:
+        first: str = ""
+        last: str = ""
+        if requester is not None:
+            first = requester.first_name
```

Comment:
**Blocking**: this violates the expected behavior and thus the AC of that ticket "When a non-CISA-admin domain manager adds a new domain manager, the email text should not try to add a first/last name of the creator and should instead show the email of whoever clicked the add user button (no check on creator)

When a cisa-admin adds a new domain manager, then the email shown should be [help@get.gov](mailto:help@get.gov). (this includes anyone with is_staff access)"

No names should be shown at all, just email address.

Reply:
Fixed!